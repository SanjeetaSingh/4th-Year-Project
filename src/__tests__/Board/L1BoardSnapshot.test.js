import renderer from 'react-test-renderer'

import Level1 from '../../Level1/Board/Board'

it('snapshot test level 1', () => {
    const tree =  renderer.create(<Level1 />).toJSON()
    expect(tree).toMatchSnapshot();
});
