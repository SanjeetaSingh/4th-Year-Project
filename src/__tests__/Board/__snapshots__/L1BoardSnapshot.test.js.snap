// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test level 1 1`] = `
<div
  className="container"
>
  <div
    className="boardcontainer"
  >
    <div
      className="board"
    >
      <div
        className="box"
        data-col={0}
        data-row={0}
      >
        <img
          alt="f"
          class="bone"
          id="food"
          src="assets/food.png"
        />
      </div>
      <div
        className="box"
        data-col={1}
        data-row={0}
      />
      <div
        className="box"
        data-col={2}
        data-row={0}
      />
      <div
        className="box"
        data-col={3}
        data-row={0}
      />
      <div
        className="box"
        data-col={4}
        data-row={0}
      />
      <div
        className="box"
        data-col={5}
        data-row={0}
      />
      <div
        className="box"
        data-col={6}
        data-row={0}
      />
      <div
        className="box"
        data-col={0}
        data-row={1}
      />
      <div
        className="box"
        data-col={1}
        data-row={1}
      />
      <div
        className="box"
        data-col={2}
        data-row={1}
      />
      <div
        className="box"
        data-col={3}
        data-row={1}
      />
      <div
        className="box"
        data-col={4}
        data-row={1}
      />
      <div
        className="box"
        data-col={5}
        data-row={1}
      />
      <div
        className="box"
        data-col={6}
        data-row={1}
      />
      <div
        className="box"
        data-col={0}
        data-row={2}
      />
      <div
        className="box"
        data-col={1}
        data-row={2}
      />
      <div
        className="box"
        data-col={2}
        data-row={2}
      />
      <div
        className="box"
        data-col={3}
        data-row={2}
      />
      <div
        className="box"
        data-col={4}
        data-row={2}
      />
      <div
        className="box"
        data-col={5}
        data-row={2}
      />
      <div
        className="box"
        data-col={6}
        data-row={2}
      />
      <div
        className="box"
        data-col={0}
        data-row={3}
      />
      <div
        className="box"
        data-col={1}
        data-row={3}
      />
      <div
        className="box"
        data-col={2}
        data-row={3}
      />
      <div
        className="box"
        data-col={3}
        data-row={3}
      />
      <div
        className="box"
        data-col={4}
        data-row={3}
      />
      <div
        className="box"
        data-col={5}
        data-row={3}
      />
      <div
        className="box"
        data-col={6}
        data-row={3}
      />
      <div
        className="box"
        data-col={0}
        data-row={4}
      />
      <div
        className="box"
        data-col={1}
        data-row={4}
      />
      <div
        className="box"
        data-col={2}
        data-row={4}
      />
      <div
        className="box"
        data-col={3}
        data-row={4}
      />
      <div
        className="box"
        data-col={4}
        data-row={4}
      />
      <div
        className="box"
        data-col={5}
        data-row={4}
      />
      <div
        className="box"
        data-col={6}
        data-row={4}
      />
      <div
        className="box"
        data-col={0}
        data-row={5}
      />
      <div
        className="box"
        data-col={1}
        data-row={5}
      />
      <div
        className="box"
        data-col={2}
        data-row={5}
      />
      <div
        className="box"
        data-col={3}
        data-row={5}
      />
      <div
        className="box"
        data-col={4}
        data-row={5}
      />
      <div
        className="box"
        data-col={5}
        data-row={5}
      >
        <img
          alt="d"
          class="image"
          id="dog"
          src="assets/dog.png"
        />
      </div>
      <div
        className="box"
        data-col={6}
        data-row={5}
      />
      <div
        className="box"
        data-col={0}
        data-row={6}
      />
      <div
        className="box"
        data-col={1}
        data-row={6}
      />
      <div
        className="box"
        data-col={2}
        data-row={6}
      />
      <div
        className="box"
        data-col={3}
        data-row={6}
      />
      <div
        className="box"
        data-col={4}
        data-row={6}
      />
      <div
        className="box"
        data-col={5}
        data-row={6}
      />
      <div
        className="box"
        data-col={6}
        data-row={6}
      />
    </div>
  </div>
  <div
    className="bottom-panel"
  >
    <div
      class="border"
    >
      <div
        class="containers"
      >
        <h2>
          Level 1:
        </h2>
        <div
          class="bubble"
        >
          Hi there! Your aim for this level is to get the dog to the food with the command butttons below. You are only allowed to use 10 commands for your sequence, so try to find the shortest path to get to the food. Once you are done with creating your sequence then go ahead and click the submit button! 
          <br />
          Good Luck!
        </div>
        <div
          class="borderPanel"
        >
          <div
            class="toppart"
          >
            <div
              class="containing"
            >
              <p
                class="number"
                data-testid="counter"
                id="count"
              >
                0/10
              </p>
            </div>
            <p
              class="titles"
            >
              Enter Sequence:
            </p>
            <p
              class="lists"
              id="action"
            />
          </div>
        </div>
        <div
          class="outside"
        >
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Left
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Up
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Down
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Right
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
             Submit
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Clear
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
